### PR TITLE
All but first column in tables are right aligned

### PR DESCRIPTION
**Pull request by shaneporter**
Fix for this issue tests for index and adds a class, `table-value`, if it is non-zero.

(I also changed the single quotes in `headline.html` to double quotes. Single are allowed, but double are used elsewhere, so better for consistency.)

### CODE REVIEW

File: _includes/components/headline.html [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
 
 <h4>Headline data</h4>
-
-<table class='table-responsive table table-hover dataTable no-footer' role='grid'>
-   <caption>{{ page.graph_title }}</caption>
+<table class="table-responsive table table-hover dataTable no-footer" role="grid">
+  <caption>{{ page.graph_title }}</caption>
   <thead>
   {% for column in sdg_headline_data[0] %}
+    {% if forloop.first %}
```
Comment (dougmet) [inline _includes/components/headline.html]: Wondering if this if is backwards. The units of measure field is not a value but this `if` statement would catch it as one?

File: _includes/components/headline.html [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
   {% for row in sdg_headline_data %}
-      <tr>
-      {% for cell in row %}
-          <td>{{ cell[1] }}</td>
-      {% endfor %}
-      </tr>
+  <tr>
+    {% for cell in row %}
+      {% if forloop.first %}
+        <td>{{ cell[1] }}</td>
```
Comment (dougmet) [inline _includes/components/headline.html]: same here. Would this not code the `Units` column as a value field?

File: _includes/components/headline.html (hunk 1 above)
Comment (shaneporter) [reply _includes/components/headline.html]: I'd forgotten about the units of measure! Thanks @dougmet :)

File: _includes/components/headline.html (hunk 2 above)
Comment (shaneporter) [reply _includes/components/headline.html]: Same as above.

File: _includes/components/headline.html (hunk 1 above)
Comment (dougmet) [reply _includes/components/headline.html]: No probs. Similar logic should work. It just needs to be the last (is there a `forloop.last`?)

Comment (shaneporter) [reply _includes/components/headline.html]: there is

**Pull request merged**